core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -54,7 +54,7 @@ private[spark] trait TaskScheduler {
   // Submit a sequence of tasks to run.
   def submitTasks(taskSet: TaskSet): Unit
 
-  // Kill all the tasks in a stage and fail the stage and all the jobs that depend on the stage.
+  // Kill all the tasks in all the stage attempts of the same stage Id
```

Comment:
Looking at the comment, shall we keep `killAllTaskAttempts` instead of `cancelTasks`, as the naming of `killAllTaskAttempts` fits the comment better?

Reply:
Updated, thanks!